packages/micropip/src/micropip/micropip.py

```diff
@@ -365,7 +381,30 @@ def install(requirements: Union[str, List[str]], keep_going: bool = False):
     )
 
 
-__all__ = ["install"]
+def list():
+    """Get the list of installed packages through micropip.
+
+    Returns
+    -------
+    ``PackageMetadataList``
+        A list of installed packages.
+
+        >>> package_list = micropip.list()
+        >>> print(package_list)
+        | Name              | Version  | Source  |
+        | ----------------- | -------- | ------- |
+        | regex             | 2021.7.6 | pyodide |
+        | tomli             | 1.2.2    | pypi    |
+        | mypy-extensions   | 0.4.3    | pypi    |
```

Comment:
Looks nice. How about,
```suggestion
        Name              | Version  | Source
        ----------------- | -------- | -------
        regex             | 2021.7.6 | pyodide
        tomli             | 1.2.2    | pypi   
        mypy-extensions   | 0.4.3    | pypi 
```
as something a bit lighter? Unless the goal was to make markdown compatible tables?